tests/yast2_gui/yast2_control_center.pm

```diff
@@ -124,10 +125,27 @@ sub start_kernel_settings {
 sub start_partitioner {
     search('partitioner');
     assert_and_click 'yast2_control-center-partitioner';
-    assert_screen 'yast2_control-center-partitioner_warning', timeout => 180;
-    send_key 'alt-y';
+    assert_screen [qw(yast2_control-center-partitioner_warning yast2_control-center-partitioner_expert)], 180;
+    # Define if storage-ng
+    set_var('STORAGE_NG', 1) if match_has_tag 'storage-ng';
+
+    if (match_has_tag 'yast2_control-center-partitioner_warning') {
+        send_key 'alt-y';
+    }
+    elsif (is_storage_ng && match_has_tag 'yast2_control-center-partitioner_expert') {
+        # Soft-fail if storage ng and no warning is shown
+        record_soft_failure 'bsc#1068905';
+    }
+    else {
+        # Fail with expected assertion in case no match
+        assert_screen 'yast2_control-center-partitioner_warning', 0;
+    }
     assert_screen 'yast2_control-center-partitioner_expert', timeout => 60;
     send_key 'alt-f';
+    if (is_storage_ng) {
+        assert_screen 'expert-partitioner-modify-confirmation';
+        wait_screen_change { send_key 'alt-o' };    #Continue
```

Comment:
that `wait_screen_change` again shouldn't be needed. We have a clear assert_screen afterwards. That's enough, isn't it?